app/src/main/java/com/itsaky/androidide/activities/MainActivity.kt

```
@@ -54,304 +56,303 @@ import com.itsaky.androidide.viewmodel.MainViewModel.Companion.SCREEN_TEMPLATE_D
+				putExtra("PROJECT_PATH", root.absolutePath)
+				addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_SINGLE_TOP)
+			}
+
+		startActivity(intent)
+	}
+
+	internal fun deleteProject(root: File) {
+		ProjectManagerImpl.getInstance().projectPath = root.absolutePath
+		try {
+			FileDeleteUtils.deleteRecursive(root)
+		} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)